modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/auth/SecondaryStoreUserLoginTestCase.java

```diff
@@ -245,4 +273,59 @@ private void stopTomcat() throws LifecycleException {
         tomcat.destroy();
         LOG.info("Tomcat server stopped.");
     }
+
+    private void addSecondaryJDBCUserStore() throws Exception {
+
+        //creating database
```

Comment:
Shall we fix the formatting here? Let's fix other places as well.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096